updater/fetchers/oracle/oracle_test.go

```diff
@@ -43,24 +46,33 @@ func TestOracleParser(t *testing.T) {
 		expectedFeatureVersions := []database.FeatureVersion{
 			{
 				Feature: database.Feature{
-					Namespace: database.Namespace{Name: "oracle:7"},
-					Name:      "xerces-c",
+					Namespace: database.Namespace{
+						Name:          "oracle:7",
+						VersionFormat: "rpm",
+					},
+					Name: "xerces-c",
 				},
-				Version: types.NewVersionUnsafe("3.1.1-7.el7_1"),
+				Version: "0:3.1.1-7.el7_1",
```

Comment:
Why changing the test? The 0 epoch is optional. Can do some tests with but some without but we should ensure it still works without.